src/modules/rewards/components/RewardsActivitiesListItem.tsx

```diff
@@ -44,48 +45,69 @@ const RewardsActivitiesListItem: FC<RewardActivitiesListItemProps> = ({ userId,
   const [errorMessage, setErrorMessage] = useState('');
 
   const RewardsActivityIcon = (type: ActvityType) => {
-
-    if (type === "follow_push_on_discord") {
-      return <Discord width={48} height={48} />
+    if (type === 'follow_push_on_discord') {
+      return <Discord width={48} height={48} />;
     }
 
     if (type === 'follow_push_on_twitter') {
-      return <Twitter width={48} height={48} />
+      return <Twitter width={48} height={48} />;
     }
 
-    // return <PushDefaultRewards />
-    return <RewardsActivity />
-    // return <Discord />
-  }
+    return <RewardsActivity />;
+  };
+
+  const resolveActivityTitle = (activityTitle: string) => {
+    const regex = /\[([^\]]+)\]\(([^)]+)\)/;
+    const match = activityTitle?.match(regex);
+    if (match) {
+      const preText = activityTitle.substring(0, match.index);
+      const linkedText = match[1];
+      const url = match[2];
+      let postText;
+      if (match.index) {
+        postText = activityTitle.substring(match.index + match[0].length);
+      }
```

Comment:
Only this logic goes to a function called getRewardsActivityTitle in /rewards/utils.tsx

```
const getRewardsActivityTitle=(title:string)=>{
    // other code
    if(match){
        return title
    }

    return null

}
```